en/SUMMARY.md

```diff
@@ -1,19 +1,20 @@
 # Summary
 
 * [Introduction](README.md)
-* [Protocol Overview](protocol/overview.md)
+* [MAVLink Versions](mavlink_version.md)
+  * [MAVLink 2](mavlink_2.md)
+* [Protocols](protocol/overview.md)
+  * [Mission Protocol](protocol/mission.md)
+  * [Parameter Protocol](protocol/parameter.md)
+  * [Command Protocol](protocol/command.md)
+  * [Camera Protocol](protocol/camera.md)
+  * [Camera Definition](protocol/camera_def.md)
+  * [Gimbal Protocol](protocol/gimbal.md)
+  * [Arm Authorization Protocol](arm_authorization.md)
```

Comment:
@zehortigoza FYI, I moved this under Protocols - hope that is OK with you.

Reply:
Makes sense to me.